2018-projects.md

```diff
@@ -0,0 +1,197 @@
+
+Future prospective improvements include but are not limited to event decisions within events, for e.g a Pepsi v/s Coke poll; Venue mapping and Uber/Lyft integration within the App; 'Wishlist' integration; option to look up and buy tickets to public events; dedicated group chats and inter-personal chat bubbles; calendar integration for appropriate reminders of events; and even the functionality of booking venues through the app, in the long run, among tons of other features. 
+
+### [CourseOn](https://github.com/PriscaJ/HackBeanpot2018.git)  ###
+
+#### Made by Prisca Joseph, Sarah Wessel, Rucha Khanolkar, and Megan Lau ####
+
+CourseOn is a web application to help students to organize which courses they need to take throughout their time at Northeastern. This allows students to generate their grad plan easily, dynamically, and interactively.
+
+### [pupDate](https://github.com/annachoi4/pupDate) ###
+
+#### Made by Anna Choi, Andres Rodriguez, Gabriella Roman, and Christopher Trinh ####
+
+A web application for dog owners to find their dogs some nearby playmates, since Anna owns a dog and is always looking for playmates! Made in PyCharm.
+----
+### [Joe & Pedro's](https://github.com/phrmsilva/HBPTweetGenerator) ###
+
+#### Made by Pedro Silva and Joe Donovan ####
+
+Our project identifies your music preferences based on how your twitter compares to those of artists' fanbases 
+
+### [Live sentiment analysis](https://github.com/divyavijaysahay/Sentiment-analysis) ###
+
+#### Made by Divyavijay Sahay, Brian, and Tram####
+
+We created a system which can perform sentiment analysis on live video and sound. This system can be used in numerous ways like for psychiatrist consultation, audience reaction and also with some more functionality blind/deaf person can use this system to analyse their surrounding.
+
+### [Chlobber](https://github.com/coderrick/Chlobber)  ###
+
+#### Made by Derrick Duncan ####
+
+Command-line utility that helps you restore your lost Chrome browsing sessions and tabs
+
+### [PaperStx](https://github.com/Jakobeha/PaperStx) ###
+
+#### Made by Jakob Hain ####
+
+An editor for block syntax, like the syntax in Scratch. It's a webpage/webapp whose custom logic is entirely client-side (e.g. you don't need a server to run it, you can just load the HTML file).
+The block syntax has basic types. Not every block can be connected to every other block. The blocks are colored, and they're have colored "holes" for other blocks - the only blocks which can go in a hole are the ones with one of the holes' colors.
+First, you create a template file. You follow a very simple DSL to either create your own block syntax or recreate an existing one (e.g. Scratch). Then, you upload this file to the webpage. Then, the webpage will give you all of the different block syntax forms, grouped by class. You can drag them onto a canvas, and create your block syntax.
+
+### [jgka](https://github.com/tylercote/spot-a-match) ###
```

Comment:
should this be removed? im confused by it

Reply:
i basically just copied and pasted the descriptions that were in the forms - a lot of them were really long so it's definitely possible to shorten them and just remove them. 

just out curiousity, what are you talking about in this case?